samples/drivers/rtc/src/main.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2024, Muhammad Waleed Badar
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/rtc.h>
+#include <zephyr/sys/util.h>
+
+const struct device *const rtc = DEVICE_DT_GET(DT_ALIAS(rtc));
+
+static int set_date_time(const struct device *rtc)
+{
+	int ret = 0;
+	struct rtc_time tm = {.tm_year = 2024 - 1900,
+			      .tm_mon = 11 - 1,
+			      .tm_mday = 17,
+			      .tm_hour = 4,
+			      .tm_min = 19,
+			      .tm_sec = 0};
```

Comment:
Nit: add a trailing comma to the last line, so the formatter would align it with one indent.

```suggestion
		.tm_sec = 0,
	};
```

Reply:
thank you for the suggestion, I have corrected the format.